src/ngSanitize/sanitize.js

```diff
@@ -207,6 +299,8 @@ function $SanitizeProvider() {
   extend = angular.extend;
   forEach = angular.forEach;
   isDefined = angular.isDefined;
+  isArray = angular.isArray;
```

Comment:
no alphabetic ordering??

Reply:
I didn't see any signs of "alphabetic ordering" 🤔 
Should I change this code?

Reply:
The `var` declarations at the top of the file are listed in alphabetical order; and @gkalpak  is usually a stickler for this kind of thing :-)

Reply:
@petebacondarwin looks like @gkalpak fixed it already

https://github.com/angular/angular.js/pull/16326/commits/19204204e57487fde16cccd02dbd21e1f70593b6#diff-699beb0d8c48a8257d3d70331bf6138dR19

@gkalpak Thanks

Reply:
Ts, ts, ts! How could you think I wouldn't have fixed that, @petebacondarwin 😛

Reply:
Fixed while merging (and also removed unused `isObject` var).